nuget.bat

```diff
@@ -0,0 +1,21 @@
+@ECHO OFF
+
+SET COMMUNITY="%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\FSharp\fsi.exe"
+SET ENTERPRISE="%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\fsi.exe"
+SET FSXSCRIPT1=scripts\cloneFsx.fsx
+SET FSXSCRIPT2=scripts\nuget.fsx
+
+IF EXIST %ENTERPRISE% (
+    SET RUNNER=%ENTERPRISE%
+) ELSE (
+    IF EXIST %COMMUNITY% (
+        SET RUNNER=%COMMUNITY%
+    ) ELSE (
+        ECHO fsi.exe not found, is F# installed?
+        EXIT /b 1
+    )
+)
+echo "Running first script"
+%RUNNER% %FSXSCRIPT1% %*
+echo "Running second script"
+%RUNNER% %FSXSCRIPT2% %*
```

Comment:
let's rather use this new version of nuget.bat: https://github.com/nblockchain/ZXing.Net.Xamarin/blob/master/nuget.bat (and include fsi.bat)